lib/game.rb

```diff
@@ -41,46 +52,84 @@ def setup
     submarine_coordinates = gets.chomp.split(',')
 
     until_valid_placement(@player.submarine, submarine_coordinates)
-
-    puts @player.board.render(true)
   end
   
   def turn
+    both_boards_rendered
+    puts "Your turn:"
     puts "Enter coord to fire on"
     coord = gets.chomp.upcase
     @player.fire_upon_computer(coord, @computer.board)
-    # The above was just so I could test the #fire_upon_computer in Player class. You can remove/update it if you want
-    
-    # At some point in here, we will call fire_upon_player(@player.board) to fire upon the player's individual cell within their board instance
-    # fire_upon_player(@player.board) should return the result of the cell being fired upon (was it a hit or miss?)
-    
-    # At some point in here, we will call fire_upon_computer(coordinate, @computer.board) the coordinate argument = the coordinate that the player chose to fire upon
-
+    # right now, both the computer and the player can win (if it comes down to the very last turn)
+    # perhaps we place a check in here to end the turn before player or computer can choose in the event they have been sunk
+    puts "My turn:"
+    @computer.fire_upon_player(@player.board)
```

Comment:
We should save the return of this to a variable. Something like `shot_result`. Then on the next line, we can do `puts "My shot was a #{shot_result}" This makes the Game class responsible for the feedback.